playbooks/roles/ses-rook/tasks/setup_for_openstack.yml

```diff
@@ -66,11 +69,36 @@
     grep 'fsid\|mon_host\|mon_initial\|network' /etc/ceph/ceph.conf | awk '{ print $3}'
   register: conf_options
 
-#- name: Get ceph conf/key file contents
-#  slurp:
-#    src: "/etc/ceph/{{ item }}"
-#  with_items:
-#    - "ceph.client.cinder.keyring"
-#    - "ceph.client.cinder_backup.keyring"
-#    - "ceph.client.glance.keyring"
-#  register: ceph_files
+- name: Get ceph conf/key file contents
+  shell:
+    set -o pipefail
+    kubectl cp rook-ceph/{{ ceph_tools_pod }}:/etc/ceph/ceph.{{ item.name }}.keyring {{ ansible_env.HOME }}/ceph.{{ item.name }}.keyring
```

Comment:
Hm, does this work for you? I think you can't do it as the pod is missing tar binary...

Reply:
I tried locally manually and it seemed to work, but as you mentioned it may not work and I just had the leftover keyrings from an old run....